worker/formatting_stack/kondo_classpath_cache.clj

```diff
@@ -8,6 +8,21 @@
   (:import
    (java.io File)))
 
+(def runner-mapping
+  {"future" `future
+   "delay"  `delay})
```

Comment:
I wanted to suggests a `defmethod`, but i guess that won't work as cleanly since it's a macro 🤔